src/main/java/duck/Option.java

```diff
@@ -0,0 +1,60 @@
+package duck;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Enumerators for different commands for clarity and consistent reference.
+ *
+ * Solution adapted from Effective Java by Joshua Bloch.
+ * https://stackoverflow.com/questions/604424/how-to-get-an-enum-value-from-a-string-value-in-java
+ */

```

Comment:
Perhaps a @see tag would make the documentation look a little bit more organised.